app/models/katello/system.rb

```diff
@@ -5,15 +5,12 @@ class System < Katello::Model
     self.include_root_in_json = false
 
     include Hooks
-    define_hooks :add_host_collection_hook, :remove_host_collection_hook,
-                 :add_activation_key_hook, :remove_activation_key_hook,
-                 :as_json_hook
+    define_hooks :as_json_hook
```

Comment:
I see it used here https://github.com/johnpmitsch/katello/blob/contenthost/app/models/katello/system.rb#L284 in the as_json method, but no where else. Remove that line from the method?